joblib/parallel.py

```diff
@@ -350,135 +347,117 @@ class Parallel(Logger):
-        values:
+        Reshaping the output when the function has several return values:
 
-        >>> from math import modf
-        >>> from joblib import Parallel, delayed
-        >>> r = Parallel(n_jobs=1)(delayed(modf)(i/2.) for i in range(10))
-        >>> res, i = zip(*r)
-        >>> res
-        (0.0, 0.5, 0.0, 0.5, 0.0, 0.5, 0.0, 0.5, 0.0, 0.5)
-        >>> i
-        (0.0, 0.0, 1.0, 1.0, 2.0, 2.0, 3.0, 3.0, 4.0, 4.0)
+        >>> from math import modf from joblib import Parallel, delayed r =
+        >>> Parallel(n_jobs=1)(delayed(modf)(i/2.) for i in range(10)) res, i =
+        >>> zip(*r) res (0.0, 0.5, 0.0, 0.5, 0.0, 0.5, 0.0, 0.5, 0.0, 0.5) i
+        >>> (0.0, 0.0, 1.0, 1.0, 2.0, 2.0, 3.0, 3.0, 4.0, 4.0)
 
         The progress meter: the higher the value of `verbose`, the more
         messages:
 
-        >>> from time import sleep
-        >>> from joblib import Parallel, delayed
-        >>> r = Parallel(n_jobs=2, verbose=5)(delayed(sleep)(.1) for _ in range(10)) #doctest: +SKIP
-        [Parallel(n_jobs=2)]: Done   1 out of  10 | elapsed:    0.1s remaining:    0.9s
-        [Parallel(n_jobs=2)]: Done   3 out of  10 | elapsed:    0.2s remaining:    0.5s
-        [Parallel(n_jobs=2)]: Done   6 out of  10 | elapsed:    0.3s remaining:    0.2s
-        [Parallel(n_jobs=2)]: Done   9 out of  10 | elapsed:    0.5s remaining:    0.1s
-        [Parallel(n_jobs=2)]: Done  10 out of  10 | elapsed:    0.5s finished
-
-        Traceback example, note how the line of the error is indicated
-        as well as the values of the parameter passed to the function that
-        triggered the exception, even though the traceback happens in the
-        child process:
-
-        >>> from heapq import nlargest
-        >>> from joblib import Parallel, delayed
-        >>> Parallel(n_jobs=2)(delayed(nlargest)(2, n) for n in (range(4), 'abcde', 3)) #doctest: +SKIP
+        >>> from time import sleep from joblib import Parallel, delayed r =
+        >>> Parallel(n_jobs=2, verbose=5)(delayed(sleep)(.1) for _ in
+        >>> range(10)) #doctest: +SKIP [Parallel(n_jobs=2)]: Done   1 out of
+        >>> 10 | elapsed:    0.1s remaining:    0.9s [Parallel(n_jobs=2)]: Done
+        >>> 3 out of  10 | elapsed:    0.2s remaining:    0.5s
+        >>> [Parallel(n_jobs=2)]: Done   6 out of  10 | elapsed:    0.3s
+        >>> remaining:    0.2s [Parallel(n_jobs=2)]: Done   9 out of  10 |
+        >>> elapsed:    0.5s remaining:    0.1s [Parallel(n_jobs=2)]: Done  10
+        >>> out of  10 | elapsed:    0.5s finished
+
+        Traceback example, note how the line of the error is indicated as well
+        as the values of the parameter passed to the function that triggered
+        the exception, even though the traceback happens in the child process:
+
+        >>> from heapq import nlargest from joblib import Parallel, delayed
+        >>> Parallel(n_jobs=2)(delayed(nlargest)(2, n) for n in (range(4),
+        >>> 'abcde', 3)) #doctest: +SKIP
```

Comment:
I broken the formatting here as well. weird.

Reply:
This is probably a side effect from the automatic line wrapper, which fails with comment formatting...